stories/TopAuctions.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+
+import { storiesOf, StoryDecorator } from '@storybook/react'
+import { object } from '@storybook/addon-knobs'
+
+import TopAuctions from 'components/TopAuctions'
+
+import { code2tokenMap } from 'globals'
+
+/*
+* ratioPairs = {
+*  'GNO/ETH': 0.31214312,
+   'OMG/ETH': 0.01976562,
+*  ...
+* }
+*/
+const ratioPairs = Object.keys(code2tokenMap).reduce((acc, code) => {
+  if (code === 'ETH') return acc
+
+  acc[`${code}/ETH`] = +Math.random().toFixed(8)
+  return acc
+}, {}) as { [pair: string]: number }
+
+
+/**
+ * sorts pairs by ASC and takes top 5
+ * @param {typeof ratioPairs} pairs 
```

Comment:
lol we really should have been documenting this way before as well - or wait I guess that's  what TS is for 😂

Reply:
Yep, I just added it after a half hour break I came back and didn't immediately understand what my code was doing